routing/probability_estimator_test.go

```diff
@@ -147,17 +153,23 @@ func TestProbabilityEstimatorMix(t *testing.T) {
 
 	// We expect the probability for a previously successful channel to
 	// remain high.
-	ctx.assertPairProbability(testTime, node1, 100, prevSuccessProbability)
+	ctx.assertPairProbability(
+		testTime, node1, 100, 0, prevSuccessProbability,
+	)
 
 	// For an untried node, we expected the node probability to be returned.
 	// This is a weighted average of the results above and the a priori
 	// probability: 0.62.
 	expectedNodeProb := (3*aprioriHopProb + 1*prevSuccessProbability) /
 		(3 + 1 + 0.25 + 0.125)
 
-	ctx.assertPairProbability(testTime, untriedNode, 100, expectedNodeProb)
+	ctx.assertPairProbability(
+		testTime, untriedNode, 100, 0, expectedNodeProb,
```

Comment:
In a later commit, you're replacing `0` by `testCapacity`. You could just as well do that in this commit and reduce the delta?

Reply:
I tried to reduce the diff as much as possible. I introduced `testCapacity` early without effect, the question is also whether we should make the probability tests depend explicitly on the `testCapacity` or only inside the assertions?

Reply:
I have no strong preference there. Commit structure looks good now.